docs/how-to/deploy-to-heroku.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+In this guide, we will be looking at how to deploy bocadillo powered app to Heroku.
+You can see example code in [heroku-example](https://github.com/bocadilloproject/heroku-example) repo.
+
+### Bocadillo Application
+
+Assuming, you have `app.py` already installed and it may look similar to this:
+
+```
+from bocadillo import App
+
+app = App()
+
+@app.route("/")
+async def index(req, res):
+    res.text = "Hello, from Heroku!"
+
+
+if __name__ == "__main__":
+    app.run()
+```
+
+Make sure it is running in local machine.
```

Comment:
Is this actually necessary when it comes to deploying to Heroku? Or did you mean that the user should verify that the app runs locally before pushing it to Heroku?